### ISSUE TITLE
Plugin error

### ISSUE THREAD

**Issue by Bui1d** [Splamy/TS3AudioBot#339]
Have problem.
My code:
```cs
using TS3AudioBot;
using TS3AudioBot.Plugins;
using TS3AudioBot.CommandSystem;

public class MyFirstPlugin : IBotPlugin
{

	public void Initialize()
	{
		//TODO
	}
	
	public void Dispose()
	{
		//TODO
	}
	
	[Command("Saytestpls")]
	public string CommandSaytestpls() {
		return "Test";
	}
}
```
TS3AudioBot error:
```text
03:07:31.9092|DEBUG Got message from xaxa: !plugin load Test.cs
03:07:31.9128| WARN Plugin "Test.cs" failed to load: System.InvalidOperationException: This bag is already loaded.
  at TS3AudioBot.CommandSystem.CommandManager.RegisterCollection (TS3AudioBot.CommandSystem.ICommandBag bag) [0x0000e] in <c43f3ea63c9c44ce81ff7b464b1a56fe>:0
  at TS3AudioBot.Plugins.Plugin.StartRegisterCommands (System.Object obj, System.Type t) [0x00030] in <c43f3ea63c9c44ce81ff7b464b1a56fe>:0
  at TS3AudioBot.Plugins.Plugin.StartInternal (TS3AudioBot.Bot bot) [0x0009c] in <c43f3ea63c9c44ce81ff7b464b1a56fe>:0
  at TS3AudioBot.Plugins.Plugin.Start (TS3AudioBot.Bot bot) [0x00047] in <c43f3ea63c9c44ce81ff7b464b1a56fe>:0
03:07:31.9141|DEBUG Command Error (Plugin error: UnknownError)

```

**Comment by Splamy** [comment]
Which version are you using?
And command names must be all lower score so: `[Command("saytestpls")]`
It seems that there is a problem in the loader where it fails to unload the plugin when it only load partially; You will have to restart the bot for now if a plugin fails to load and you want to retry.

**Comment by Splamy** [comment]
Fixed in 63dc0ee2fcf3eb498a33d4c65db00a583c690ca8
note from previous comment still stands.
